tool/tele/cli/commands.go

```diff
@@ -50,11 +54,45 @@ type VersionCmd struct {
 	Output *constants.Format
 }
 
+// HelmCmd combines commands operating on helm charts.
+type HelmCmd struct {
+	*kingpin.CmdClause
+}
+
+// HelmBuildCmd builds an application image out of a helm chart.
+type HelmBuildCmd struct {
+	*kingpin.CmdClause
+	// Path is the path to a helm chart.
+	Path *string
+	// OutFile is the output tarball file
+	OutFile *string
+	// Overwrite overwrites existing tarball
+	Overwrite *bool
+	// VendorPatters is file pattern to search for images
+	VendorPatterns *[]string
+	// VendorIgnorePatterns if file pattern to ignore when searching for images
+	VendorIgnorePatterns *[]string
+	// SetImages rewrites images to specified versions
+	SetImages *loc.DockerImages
+	// Parallel defines the number of tasks to execute concurrently
+	Parallel *int
+	// Quiet allows to suppress console output
+	Quiet *bool
+	// Verbose enables more detailed build output.
+	Verbose *bool
```

Comment:
Why not use `--debug` for this? `tele helm --debug build --verbose` would look confusing.